src/Lean/Elab/BuiltinCommand.lean

```diff
@@ -138,6 +176,86 @@ private partial def elabChoiceAux (cmds : Array Syntax) (i : Nat) : CommandElabM
     aliases := aliases.push (currNamespace ++ id, declName)
   modify fun s => { s with env := aliases.foldl (init := s.env) fun env p => addAlias env p.1 p.2 }
 
+/-- Brings some definitions in scope.
+
+* `open Some.Namespace.Ident` brings all definitions in `Some.Namespace.Ident` in scope.
+* `open Some.Namespace.Ident (def1 def2)` only brings `Some.Namespace.Ident.def1` and
+  `Some.Namespace.Ident.def2` in scope.
+* `open Namespace renaming def1 → def1', def2 → def2'` brings `def1` and `def2` in scope under the
```

Comment:
Is there any way to have a specialized docstring show up for hovers on the "renaming" keyword?

Reply:
Not as far as I know, no :/